static/app/components/events/eventStatisticalDetector/spanDiff.tsx

```diff
@@ -0,0 +1,227 @@
+import {useTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import {DataSection} from 'sentry/components/events/styles';
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import TextOverflow from 'sentry/components/textOverflow';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconAdd, IconSubtract} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
+import {getDuration} from 'sentry/utils/formatters';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {spanDetailsRouteWithQuery} from 'sentry/views/performance/transactionSummary/transactionSpans/spanDetails/utils';
+
+type SpanDiff = {
+  duration_after: number;
+  duration_before: number;
+  duration_delta: number;
+  freq_after: number;
+  freq_before: number;
+  freq_delta: number;
+  sample_event_id: string;
+  score_delta: number;
+  span_description: string;
+  span_group: string;
+  span_op: string;
+};
+
+type DiffRowProps = {
+  after: number;
+  before: number;
+  delta: number;
+  end: string;
+  group: string;
+  label: string;
+  op: string;
+  projectId: string;
+  start: string;
+  transaction: string;
+};
+
+type UseFetchAdvancedAnalysisProps = {
+  breakpoint: string;
+  end: string;
+  projectId: string;
+  start: string;
+  transaction: string;
+};
+
+function useFetchAdvancedAnalysis({
+  transaction,
+  start,
+  end,
+  breakpoint,
+  projectId,
+}: UseFetchAdvancedAnalysisProps) {
+  const organization = useOrganization();
+  return useApiQuery<SpanDiff[]>(
+    [
+      `/organizations/${organization.slug}/events-root-cause-analysis/`,
```

Comment:
before we GA, we should try to change the name from `root-cause` to `change-analysis` or something like that to ensure consistency with how we've been referring to this feature internally